src/hyp3_isce2/process.py

```diff
@@ -2,39 +2,71 @@
 ISCE2 processing
 """
 
-import argparse
 import logging
+import os
+import subprocess
+from argparse import ArgumentDefaultsHelpFormatter, ArgumentParser
 from pathlib import Path
 
 from hyp3_isce2 import __version__
+from hyp3_isce2.burst import (BurstParams, download_bursts,
+                              get_region_of_interest)
 
 log = logging.getLogger(__name__)
 
+TOPSAPP = str(Path(os.getenv('ISCE_HOME')) / 'applications' / 'topsApp.py')
 
-def process_isce2(greeting: str = 'Hello world!') -> Path:
-    """Create a greeting product
+
+def topsapp_burst(
+    reference_scene: str,
+    secondary_scene: str,
+    swath_number: int,
+    reference_burst_number: int,
+    secondary_burst_number: int,
+    polarization: str = 'VV',
+    azimuth_looks: int = 4,
+    range_looks: int = 20,
+) -> None:
+    """Create a burst interferogram
 
     Args:
-        greeting: Write this greeting to a product file (Default: "Hello world!" )
+        reference_scene: Reference SLC name
+        secondary_scene: Secondary SLC name
+        swath_number: Number of swath to grab bursts from (1, 2, or 3) for IW
+        reference_burst_number: Number of burst to download for reference (0-indexed from first collect)
+        secondary_burst_numbe: Number of burst to download for secondary (0-indexed from first collect)
+        polarization: Polarization to use
+        azimuth_looks: Number of azimuth looks
+        range_looks: Number of range looks
     """
-    log.debug(f'Greeting: {greeting}')
-    product_file = Path('greeting.txt')
-    product_file.write_text(greeting)
-    return product_file
+    ref_params = BurstParams(reference_scene, f'IW{swath_number}', polarization.upper(), reference_burst_number)
+    sec_params = BurstParams(secondary_scene, f'IW{swath_number}', polarization.upper(), secondary_burst_number)
+    ref_metadata, sec_metadata = download_bursts([ref_params, sec_params])
+
+    is_ascending = ref_metadata.orbit_direction == 'ascending'
+    insar_roi = get_region_of_interest(ref_metadata.footprint, sec_metadata.footprint, is_ascending=is_ascending)
+    dem_roi = ref_metadata.footprint.intersection(sec_metadata.footprint).bounds
+    print(insar_roi, dem_roi)
+    subprocess.run(['python', TOPSAPP, '-h'])
```

Comment:
Hmm, I'm not sure I understand what's going on here. What's the reason for calling `TOPSAPP` with just the `-h` arg?

Reply:
This is a placeholder to prove that we *can* call topsApp in this spot. This will be replaced with the actual processing call once we have the functionality for downloading the input data set up.

Reply:
Thanks for adding it!